<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PopStarGame</class>
 <widget class="QDialog" name="PopStarGame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_StartGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GameInstruction">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>348</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_GameVersion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>402</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ExitGame">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>459</y>
     <width>180</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
